src/views/IndexView.vue

```diff
@@ -11,15 +11,16 @@ const { issues, isLoading, isFinalPageFetched, total, loadFlaws, loadMoreFlaws }
 const tableFilters = ref<Record<string, string>>({});
 
 const showFilter = computed(() =>
-  Object.keys(searchStore.searchFilters).length > 0
+  Object.keys(searchStore.searchFilters).length > 0 || searchStore.queryFilter !== ''
 );
 
 const isDefaultFilterSelected = ref<boolean>(showFilter.value);
 
 const params = computed(() => {
   const paramsObj = {
     ...tableFilters.value,
-    ...isDefaultFilterSelected.value && searchStore.searchFilters
+    ...isDefaultFilterSelected.value && searchStore.searchFilters,
+    ...isDefaultFilterSelected.value && { query: searchStore.queryFilter }
```

Comment:
We need to merge #394 to avoid this kind of thing 😆 
```suggestion
    ...isDefaultFilterSelected.value && { query: searchStore.queryFilter },
```